tests/chainer_tests/optimizer_hooks_tests/test_gradient_hard_clipping.py

```diff
@@ -3,54 +3,90 @@
 import numpy as np
 
 import chainer
-from chainer import backend
-import chainer.initializers as I
 from chainer import optimizer_hooks
 from chainer import optimizers
 from chainer import testing
-from chainer.testing import attr
+
+_backend_params = [
+    # NumPy
+    {},
+    {'use_ideep': 'always'},
```

Comment:
The test seems to fail in use of iDeep.

Reply:
Have done a fix but not really sure if it is the way to go. Please review.